frontend/src/lib/StaticGraphParser.ts

```diff
@@ -52,214 +48,105 @@ export class SelectedNodeInfo {
   }
 }
 
-export function _populateInfoFromTemplate(
```

Comment:
this function is argo specific - will need to get a generic function to handle both